packages/block-library/src/query/utils.js

```diff
@@ -375,7 +379,22 @@ export const useUnsupportedBlocks = ( clientId ) => {
 			getClientIdsOfDescendants( clientId ).forEach(
 				( descendantClientId ) => {
 					const blockName = getBlockName( descendantClientId );
-					if ( ! blockName.startsWith( 'core/' ) ) {
+					/*
+					 * Client side navigation can be true in two states:
+					 *  - supports.interactivity = true;
+					 *  - supports.interactivity.clientNavigation = true;
+					 */
+					const blockInteractivityBool = Object.is(
+						getBlockSupport( blockName, 'interactivity' ),
+						true
+					);
+					const blockClientNavigation = getBlockSupport(
```

Comment:
Can we rename this to a more descriptive name? Something like `blockSupportsInteractivityClientNavigation`